server/routes/send.js

```diff
@@ -13,29 +14,21 @@ module.exports = function({logger, licenceService, authenticationMiddleware}) {
     });
 
     router.get('/:nomisId', asyncMiddleware(async (req, res) => {
-        logger.debug('GET /send');
 
-        res.render('send', {nomisId: req.params.nomisId});
-    }));
-
-    router.post('/omu/', asyncMiddleware(async (req, res) => {
-        logger.debug('POST /send/omu');
-
-        const nomisId = req.body.nomisId;
+        const {nomisId} = req.params;
+        const licence = await licenceService.getLicence(nomisId, {populateConditions: true});
```

Comment:
Probably don't need to populate conditions do you?

Reply:
ta. Copy error.